pkg/porter/install.go

```diff
@@ -32,6 +35,14 @@ func (p *Porter) InstallBundle(opts InstallOptions) error {
 		return err
 	}
 
+	if len(opts.CredentialIdentifiers) == 0 {
```

Comment:
I'd vote to add this same logic to the other commands (upgrade, uninstall, custom action)... I think @carolynvs was mentioning the same?